toolkits/landcover/impl/bands.js

```diff
@@ -146,9 +148,42 @@ function addFractionalYearBand(collection) {
   });
 }
 
+
+/**
+ * Compute a matrix multiply between the given image and a set of coefficients
+ * such as an eigenvector rotation or a Tasseled Cap transformation.
+ *
+ * The input image is expected to have the same number of bands as the width of
+ * the coefficients matrix.
+ *
+ * @param {!ee.Image} image The image on which to compute the transformation.
+ * @param {!Array<Array<number>>} coef The matrix to apply to the image.
+ * @param {?Array<string>} bandNames An optional list of output band names.
+ * @return {!ee.Image} The transformed bands.
+ */
```

Comment:
Please add unit tests to tests/unit/bands.test.unit.js (sending out #25 with skeleton today)

Reply:
Done

Reply:
I don't see that file, only integration tests in tests/int/bands.test.int.js.

I realized after writing my last comment that there are a bunch of other unit tests that are missing, so I'll add them for other existing logic follow-up PRs.

If you haven't started writing unit tests for this yet, you can find a simple example here: https://github.com/google/earthengine-community/blob/be248b7ca4c9fe3a911212f87f11f291534d7010/toolkits/landcover/test/unit/landsat8.unit.test.js#L18-L33

Reply:
I don't see what a meaningful unit test for this looks like.

As we've discussed several times, repeating the computation calls here isn't helpful and will be brittle.

The only thing meaningful I see to do is just make the call and verify that it returns a ComputedObject.

Reply:
I understood both you and @tylere felt strongly that we *should* have unit tests, and I agreed. The outcome of those discussions are documented in https://github.com/google/earthengine-community/blob/landcover-toolkit/toolkits/landcover/test/README.md. Please let me know if you want to revisit, or If you prefer I can take a stab at writing a unit tests for this.

Reply:
Sgtm.